packages/test/test-end-to-end-tests/src/test/dataStoresNested.spec.ts

```diff
@@ -78,9 +70,9 @@ describeCompat("Nested DataStores", "NoCompat", (getTestObjectProvider, apis) =>
 
 	async function addContainer(container: IContainer) {
 		containers.push(container);
-		const dataStores = (await container.getEntryPoint()) as IDataStores;
+		const channelCollection = (await container.getEntryPoint()) as ChannelCollection;
```

Comment:
Can we use here newly added interface, instead of concrete class type?

Reply:
We need some other methods from other interfaces but I'll make my own ad-hoc in the test so we don't bind to the concrete class..